src/crafting.cpp

```diff
@@ -326,7 +326,7 @@ float Character::crafting_speed_multiplier( const item &craft,
 
     // If we're working below 20% speed, just give up
     if( total_multi <= 0.2f ) {
-        add_msg_if_player( m_bad, _( "You are too frustrated to continue and just give up." ) );
+        add_msg_if_player( m_bad, _( "Your progress is so slow you give up in frustration." ) );
```

Comment:
```suggestion
        add_msg_if_player( m_bad, _( "Your progress is so slow so you give up in frustration." ) );
```

Reply:
I think "Your progress is so slow **that** you give up in frustration." works better than "Your progress is so slow so you give up in frustration."?

Either that or "Your progress is slow so you give up in frustration." works too.

Reply:
Updated to "that" variant.